docs/configuration/worker-configuration.md

```diff
@@ -66,12 +66,12 @@ All parameters are described in the [Worker parameters](#worker-parameters) sect
 You can adjust worker pods in two ways: set fixed value in `replicaCount`,
 or enable `autoscaling`, which scales pods automatically. 
 
-#### Real life scenario: I use SC4SNMP for only trap monitoring, I want to use my resources effectively.
+#### Real life scenario: I use SC4SNMP for only trap monitoring, and I want to use my resources effectively.
 
 If you don't use polling at all, set `worker.poller.replicaCount` to `0`.
-If you'll want to use polling in the future, you need to increase `replicaCount`. To monitor traps, adjust `worker.trap.replicaCount` depending on your needs and `worker.sender.replicaCount` to send traps to Splunk. Usually you need much less sender pods than trap ones.
+If you want to use polling in the future, you need to increase `replicaCount`. To monitor traps, adjust `worker.trap.replicaCount` depending on your needs and `worker.sender.replicaCount` to send traps to Splunk. Usually, you need significantly fewer sender pods than trap pods.
 
-This is the example of `values.yaml` without using autoscaling:
+This following is an example of `values.yaml` without using autoscaling:
```

Comment:
I think we can remove word this from here.

Reply:
"This" should be replaced with "that"